test/test_type_hints.py

```diff
@@ -170,6 +170,30 @@ def test_doc_examples(self):
             except subprocess.CalledProcessError as e:
                 raise AssertionError("mypy failed.  Look above this error for mypy's output.")
 
+    @unittest.skipIf(sys.version_info[0] == 2, "no type hints for Python 2")
+    @unittest.skipIf(not HAVE_MYPY, "need mypy")
+    def test_type_hint_examples(self):
+        """
+        Runs mypy over all the test examples present in
+        `type_hint_tests` directory.
+        """
+    test_path = os.path.dirname(os.path.realpath(__file__))
+    examples_folder = os.path.join(test_path, "type_hint_tests")
+    examples = os.listdir(examples_folder)
+    for example in examples:
+        example_path = os.path.join(os.path.dirname(os.path.realpath(__file__)),
```

Comment:
This is redundant now (you've defined example_path below as well)

Reply:
Fixed